.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "workbench.panel.defaultLocation": "bottom"
+}
```

Comment:
Be careful with what you commit, `git status` will show you the files waiting to be committed. This is a personal developer preference, so `.vscode` should be added to the `.gitignore` file